1-exercises/C-while-loop-with-array/exercise.js

```diff
@@ -18,6 +18,14 @@ const BIRTHDAYS = [
 
 function findFirstJulyBDay(birthdays) {
     // TODO
+    let i = 0;
+    while (i < birthdays.length) {
+          if (birthdays[i] === "July 11th"){
```

Comment:
1. indent back
2. There is also a function to check the beginning of the string in js to match with "July" only. Google it as an exercise